_layouts/default.html

```diff
@@ -12,6 +12,7 @@
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
 
     <link href="{{ site.baseurl }}/css/style.css" rel="stylesheet">
+    <title>{{ title }}</title>
```

Comment:
It has to be `{{ page.title }}`. For some reason I cannot suggest the change here.